splunk_setup/splunk.sh

```diff
@@ -0,0 +1,527 @@
+#!/bin/bash
```

Comment:
It would be great to write a log file with timestamps on the actions that were taken.